modules/icgc-argo-workflows/payload/qcmetrics/main.nf

```diff
@@ -10,30 +10,34 @@ process PAYLOAD_QCMETRICS {
         'quay.io/biocontainers/multiqc:1.13--pyhdfd78af_0' }"
 
     input:  // input, make update as needed
-      tuple val(meta), path(metadata_analysis), path(files_to_upload), path(multiqc)
-      path pipeline_yml
+        tuple val(meta), path(files_to_upload), path(metadata_analysis), path(multiqc)
+        path pipeline_yml
+        val genome_build
+        val genome_annotation
```

Comment:
I'm not sure on the benefit of making these mandatory arguments rather than optional.

Additionally, presumably they're already passed in via `meta`

Reply:
Yes, I agree with @edsu7, it is better to make them optional arguments.

Reply:
Updates: 
1. genome_build and genome_annotation to be provided by meta, no longer required as mandatory arguments.
2. genome_annotation is optional for payload alignment module
3. both genome_annotation and genome_build are optional for payload qc metrics module.